apps/web/src/pages/Profile/CareerTimelinePage/components/CareerTimelineSection.tsx

```diff
@@ -60,21 +60,14 @@ const CareerTimelineSection = ({
 
   return (
     <>
-      <div
-        data-h2-flex-grid="base(center, x1, x1)"
-        data-h2-padding-bottom="base(x.5)"
-      >
+      <div className="mb-3 flex flex-col justify-between gap-6 sm:flex-row sm:items-end">
```

Comment:
The filters are not lining up with the add link anymore. As well as the padding on the left side isn't lining up with the content anymore. 

https://www.chromatic.com/review?appId=61e099a1bb1465003a73d3ce&number=13780&type=linked&activeElementId=comparison-test-key-662a74d55ee9411d85b6634c-1200px&view=changes